cookies/meta-blocked.tentative.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<head>
+  <meta http-equiv="set-cookie" content="meta-set-cookie=1">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
```

Comment:
Can you link to https://github.com/whatwg/html/issues/3076 in the file itself? Over time, it can become more difficult to track down the original commit when blame get confused by multiple similar tests being renamed, or when one doesn't know about `git log --follow`.